src/index.html

```diff
@@ -11,12 +11,54 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="Logo"
+          class="logo"
+        />
+      </a>
+      <nav>
+        <ul class="menu">
+          <li>
+            <a
+              href="#apple"
+              class="is-active"
+            >
+              <span class="highlight">Apple</span>
+            </a>
+          </li>
+          <li><a href="#samsung">Samsung</a></li>
```

Comment:
```suggestion
          <li class="menu__item">
            <a 
              href="#samsung" 
              class="menu__link"
            >
              Samsung
            </a>
          </li>
```
Consider adding classes to elements